### ISSUE TITLE
Sit Bug 1.19.4

### ISSUE THREAD

**Issue by Leon-CZ** [Zrips/CMI#7699]
### Description of Issue

If I sit down on the stairs and then stand up using the left shift, I can't sit down again, the plugin thinks I'm still sitting.

### Version Information

```shell
https://cdn.discordapp.com/attachments/526402563847880725/1088517023480422440/image.png
```

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by mrfloris** [comment]
On Paper 1.19.3 I can sit, sneak unsit, .. and sit again.
On Paper 1.19.4, I can sit, sneak unsit, but then I can't sit again. 

So I guess I can confirm this is an issue on 1.19.4 

I get no console msgs, and cmilib/cmi are currently up to date, and I've bumped my paper version to the latest
```
> cmi version
[19:05:05 INFO]: --------------------------------------------------
[19:05:05 INFO]: CMI: 9.4.0.2 SqLite
[19:05:05 INFO]: CMILib: 1.2.5.2
[19:05:05 INFO]: Server: Paper(466) 1.19.4-R0.1-SNAPSHOT
[19:05:05 INFO]: CMI economy: Enabled Vault: 1.7.3-CMI CMI Chat: Enabled
[19:05:05 INFO]: Modules -> 56 enabled 1 disabled: votifier
[19:05:05 INFO]: --------------------------------------------------
>
```

**Comment by mrfloris** [comment]
This is fixed in CMI 9.5.0.x
